### PR TITLE
Fix multipart/form-data aka file uploads

### PR DESCRIPTION
**Pull request by analogue**
Fix for #113 and probably #99 too.

Will update changelog, version, etc when PR gets the thumbs up.

### CODE REVIEW

File: pyramid_swagger/load_schema.py [hunk 1]
```diff
@@ -88,6 +88,15 @@ def swagger_type_validator(validator, ref, instance, schema):
     return swagger_type_validator
 
 
+def type_validator(validator, types, instance, schema):
+    """Swagger 1.2 supports parameters of 'type': 'File'. Skip validation of
```
Comment (jnb) [inline pyramid_swagger/load_schema.py]: If you don't special-case the 'File' validation, and instead just use the general-case validation, does it fail?

File: pyramid_swagger/load_schema.py [hunk 2]
```diff
@@ -124,6 +133,7 @@ def get_body_validator(models):
     {
         'paramType': ignore,
         'name': ignore,
+        'type': type_validator,
```
Comment (jnb) [inline pyramid_swagger/load_schema.py]: I can't see where this field is used?

File: pyramid_swagger/load_schema.py (hunk 1 above)
Comment (analogue) [inline pyramid_swagger/load_schema.py]: Yes, ```test_200_with_file_upload(test_app)``` fails unless the special case is handled.

File: pyramid_swagger/load_schema.py (hunk 2 above)
Comment (analogue) [inline pyramid_swagger/load_schema.py]: https://github.com/striglia/pyramid_swagger/blob/master/pyramid_swagger/load_schema.py#L146

**Comment by jnb** [comment]
:+1:

**Comment by analogue** [comment]
This is ready to merge. I updated the change log but left the version unchanged because I have another PR (separate bug) that needs to make it into the next release candidate.

**Pull request merged**